__tests__/animation/fluidResistance.spec.ts

```diff
@@ -162,7 +162,7 @@ describe('fluidResistance', () => {
     const mockElements: AnimatingElement<
       FluidResistanceConfig,
       HTMLElement
-    >[] = new Array(3).fill({ ...baseElement, repeat: 3 });
+    >[] = new Array(3).fill({ ...baseElement, repeat: 2 });
```

Comment:
These new tests now better reflect `renature`'s actual inner workings. The first run (initial animation) should set `state.repeatCount` to 0, so that all subsequent animations begin incrementing the repeat count.